test/util/test_runners/test_runner_args.py

```diff
@@ -0,0 +1,44 @@
+from argparse import ArgumentParser
+
+
+class TestRunnerArgs:
+    """
+    Dto to hold the tests environment arguments
+    """
+
+    def __init__(self, hostname, account, username, password, invoke_cli_as_process, working_cli_path_only_for_init: str,
+                 cli_to_test_path: str, files_folder: str, test_name_identifier="integration"):
+        self.hostname = hostname
+        self.account = account
+        self.username = username
+        self.password = password
+        self.test_name_identifier = test_name_identifier
+        self.invoke_cli_as_process = invoke_cli_as_process
+        self.working_cli_path_only_for_init = working_cli_path_only_for_init
+        self.cli_to_test_path = cli_to_test_path
+        self.files_folder = files_folder
+        if cli_to_test_path is not None and  working_cli_path_only_for_init.lower() == cli_to_test_path.lower():
+            raise RuntimeError("cli to start enviroment and tested cli are on the same path")
+
+    @staticmethod
+    def create_from_args():
+        parser = ArgumentParser(description='Test arguments')
+
+        # Add the arguments
+        parser.add_argument('-invoke_cli_as_process', action='store_true',
+                            help='if using it, unitests will run cli process. otherwise it will run python code')
+        parser.add_argument('-i', '--identifier', dest='test_name_identifier', action='store', default='integration',
```

Comment:
"If added, integration tests will run as a process executable. Otherwise it will run as code, requiring Python"